instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/attributes.go

```diff
@@ -56,11 +56,18 @@ func RequestIDAttr(requestID string) attribute.KeyValue {
 
 // DefaultAttributeSetter checks to see if there are service specific attributes available to set for the AWS service.
 // If there are service specific attributes available then they will be included.
+// Deprecated: Kept for backward compatibility, use DefaultAttributeBuilder instead. This will be removed in a future release.
```

Comment:
please do the same in other places

```suggestion
//
// Deprecated: Use DefaultAttributeBuilder instead. This will be removed in a future release.
```

![image](https://github.com/user-attachments/assets/db937fb0-bfb7-4c9c-9d0b-8937059ba77b)